resources/assets/js/services/ProducingActivityProcessExporter.js

```diff
@@ -59,14 +60,26 @@ export default class ProducingActivityProcessExporter {
         this.activities.forEach((activity, index) => {
             let lines = headers.map(header => {
                 if (unwantedColumns.indexOf(header) !== -1) return;
-                if(header === 'description') {
+                if (header === 'description') {
                     return _.capitalize(exportTranslatedFieldMapping[header]) + ": \n\t" + activity[header] + " \n";
                 }
                 if (activity[header] === null || activity[header] === 'null') {
                     return _.capitalize(exportTranslatedFieldMapping[header]) + ": -";
                 }
-                if(header === 'feedback') {
-                    return _.capitalize(exportTranslatedFieldMapping[header]) + ": " + this.getFeedbackUrl(activity[header]);
+                if (header === 'feedback') {
+                    if (this.includeReflections) {
+                        return _.capitalize(exportTranslatedFieldMapping[header]) + ": \n\t" +
+                            "Waarom is dit moeilijk?:" + (activity[header]['notfinished'] ? `${activity[header]['notfinished']}.` : "-") + "\n\t" +
```

Comment:
These strings, e.g. "Hulp gekregen", should also be translated, otherwise an English export will have Dutch labels.